### ISSUE TITLE
IOS Mobile App - Send Button Missing When Uploading Pictures

### ISSUE THREAD

**Issue by riise317** [RocketChat/Rocket.Chat.ReactNative#4786]
### Description:

When user has "Prefer Cross-Fade Transitions" on under Settings --> Accessibility --> Motion, they are unable to upload pictures.  Once they click the + and choose a picture, the bottom bar with the comment section and send button never appear.

### Environment Information:

- Rocket.Chat Server Version: 5.0.6
- Rocket.Chat App Version:4.34.0
- Device Name: 
- OS Version: 16.1.1

### Steps to reproduce:

1.  Settings, Accessibility, Motion
2.  Prefer Cross-Fade Transitions

### Expected behavior:

 Send button should appear when attempting to upload a picture

### Actual behavior:

 No send button

### Additional context
![without_prevent_crossfade_transitions](https://user-images.githubusercontent.com/121652560/210032507-7ba2c74d-9fb2-4d2d-bd72-1adbc6e83727.png)
![with_prevent_crossfade_transitions](https://user-images.githubusercontent.com/121652560/210032508-dd8ee6fa-5818-4c61-9f17-a23022d7c243.png)

**Comment by diegolmello** [comment]
Duplicated https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/3846
